.github/workflows/ci.yml

```diff
@@ -898,7 +898,7 @@ jobs:
 
       - uses: actions/setup-python@v5
         with:
-          python-version: "3.13"
+          python-version: "3.13b1"
```

Comment:
I think this is what you need:

```suggestion
          python-version: "3.13.0-beta.1"
```

Reply:
(It needs to match one of the versions in the manifest file at https://github.com/actions/python-versions/blob/main/versions-manifest.json)